projects/ubuntu-22.04/Dockerfile

```diff
@@ -1,36 +1,39 @@
 FROM ubuntu:22.04
 
-RUN \
-  apt-get update && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    wget \
-    apt-transport-https \
-    libfuse2 \
-    unixodbc \
-    rsyslog \
-    netcat \
-    gnupg \
-    lsb-release \
-    python3 \
-    python3-distro \
-    python3-psutil \
-    python3-jsonschema \
-    python3-requests \
-    python3-pip \
-    libcurl4-gnutls-dev \
-    unixodbc-dev \
-  && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* /tmp/*
-
-RUN pip3 install xmlrunner distro
-
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN apt-get update && \
+    apt-get install -y \
+        apt-transport-https \
+        gnupg \
+        wget \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
 #RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
-    #echo "deb [arch=amd64] https://packages.irods.org/apt/ jammy main" | tee /etc/apt/sources.list.d/renci-irods.list
+    #echo "deb [arch=amd64] https://packages.irods.org/apt/ focal main" | tee /etc/apt/sources.list.d/renci-irods.list
```

Comment:
Oops

These will need to be uncommented in the future when we have published externals packages, anyway

Reply:
And this is just a note to yourself to do that once everything aligns, correct?

Reply:
Yeah, once we have packages for jammy these will need to be uncommented. Also I accidentally changed the jammy to a focal here. :/